WCDB.swift.podspec

```diff
@@ -25,13 +25,13 @@ Pod::Spec.new do |wcdb|
     "GCC_PREPROCESSOR_DEFINITIONS" => "SQLITE_HAS_CODEC WCDB_BUILTIN_SQLCIPHER",
     'SWIFT_WHOLE_MODULE_OPTIMIZATION' => 'YES',
     'APPLICATION_EXTENSION_API_ONLY' => 'YES',
-    "HEADER_SEARCH_PATHS" => "${PODS_ROOT}/WCDBSwift",
+    "HEADER_SEARCH_PATHS" => "${PODS_ROOT}/WCDB.swift",
```

Comment:
This one should use `WCDB.swift`. CocoaPods does not use your `module_name` as folder name. It just use the `podspec.name`